app/microservice/register.json

```diff
@@ -21,6 +21,16 @@
         }
       ]
     },
+    {
+      "url": "/v1/geostore/multiple",
```

Comment:
for consistency, call this "find-by-ids" -> https://github.com/resource-watch/layer/blob/develop/app/microservice/register.json#L67